nextflow.config

```diff
@@ -1,17 +1,24 @@
 /*
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    nf-core/metaval Nextflow config file
+    genomic-medicine-sweden/meta-val Nextflow config file
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     Default config options for all compute environments
 ----------------------------------------------------------------------------------------
 */
 
+nextflow.enable.strict = true
+
 // Global default params, used in configs
 params {
 
     // TODO nf-core: Specify your pipeline's command line flags
     // Input options
     input                      = null
+    taxid                      = null
+    pipelines_testdata_base_path = "https://raw.githubusercontent.com/nf-core/test-datasets/createtaxdb"
```

Comment:
Not sure why this is needed here.

Reply:
It's a test viral pathogen genome that will be used later.

Reply:
<img width="193" alt="Screenshot 2024-06-03 at 20 05 23" src="https://github.com/genomic-medicine-sweden/meta-val/assets/91951607/8f3aabb6-3741-4071-a3f0-95cc04a504c2">


 I get this, is the url correct?

Reply:
Here is the usage: https://github.com/genomic-medicine-sweden/meta-val/blob/1ea35aaa6ffb8d69e9fe7bf2e774efd09eb57daa/nextflow.config#L19